### PR TITLE
go-ethereum: Update to 1.8.19

### PR DESCRIPTION
**Pull request by sihoang**

### CODE REVIEW

File: srcpkgs/go-ethereum/template [hunk 1]
```diff
@@ -12,13 +12,12 @@ ${go_import_path}/cmd/puppeth
 ${go_import_path}/cmd/rlpdump
 ${go_import_path}/cmd/wnode
 ${go_import_path}/cmd/swarm"
-depends="geth"
```
Comment (sihoang) [inline srcpkgs/go-ethereum/template]: @maxice8 do we need this?
The go-package already includes `${go_import_path}/cmd/geth`

Comment (maxice8) [reply srcpkgs/go-ethereum/template]: Yes we need that go-etherum depend on geth, we have split geth into a subpackage so the geth binary isn't in the main package.

Comment (sihoang) [reply srcpkgs/go-ethereum/template]: that makes sense.

Comment (sihoang) [reply srcpkgs/go-ethereum/template]: got it. Thanks for the explanation. I've fixed the PR.

**Pull request merged**